app/policies/comment_policy.rb

```diff
@@ -0,0 +1,32 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :current_user, :comment
+
+  def initialize(current_user, comment)
+    @current_user = current_user
+    @comment = comment
+  end
+
+  def show?
+    true
+  end
+
+  def new?
+    true
+  end
+
+  def create?
+    true
+  end
+
+  def edit?
+    current_user == comment.author
```

Comment:
Do we need this for both edit and update? I went with doing one or the other, as making the policy for either seemed to work. Not sure if that was right or not though!